src/main/java/BuyStockUseCase/PortfolioPresenter.java

```diff
@@ -9,5 +9,15 @@ public class PortfolioPresenter {
     public PortfolioPresenter(iPortfolioGUI view, Portfolio portfolio) {
         this.view = view;
         this.portfolio = portfolio;
+        this.view.addBackAction(this::onBack);
+        this.view.addSearchAction(this::onSearch);
+    }
+    public void onBack() {
+        view.close();
+    }
+    public void onSearch() {
+        // TODO: CONNECT THE SEARCH FEATURE
+        String symbol = view.getSearchField();
```

Comment:
Once I merge the search stock GUI I will connect this function to the controller so that it can redirect to the search stock page. For now it's fine, but once I connect it to the search stock GUI a try catch needs to be added in case the String symbol is not valid symbol, and a message needs to be displayed to the user. I will add this feature later but just want to make note of it.